pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Batch Operations"
+permalink: batch-ops-2.1.html
+folder: samvera/manager_guide/batch-works-2.1.md
+sidebar: home_sidebar
+a-z: ['Batch Operations - Repository Administration']
+keywords: Best Practices, managers, repo mangers, hyrax administration
+tags: [feature_guide]
+categories: How to use the Administration panel in hyrax
+version:
+  id: 'hyrax_2.1-stable'
+  versions:
+  - label: 'Hyrax 1.0'
+    link:  'batch-works-1.0.html'
+  - label: 'Hyrax 2.0'
+    link:  'batch-works-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'batch-ops-2.1.html'
+    selected: 'true'
+---
+## Uploading Works
+Batch upload lets you upload multiple files at once as separate works. This is useful when you have several works, where each work is a single file, and each work has the same creator, such as technical reports from a organization or photographs by an artist.
+
+To create works as a batch, go to your Dashboard and click on `Works` in the sidebar. There, you will see a table of your works and options to `Add new work` or `Create batch of works`.
+
+![Create batch of works](/images/screenshots/create-batch-works-1.png)
+
+If you have more than one work type, you will be prompted to select the work type. You can only create a batch of works of the same work type.
+
+### Adding Files to Works
+The “Files” tab is for uploading files, either locally or from a cloud service. This works in the same way as adding files to a new work. You can upload one file at a time, or select multiple files.
+
+![Upload files](/images/screenshots/create-batch-works-2.png)
+
+After uploading files, you can edit the `Display Label` and `Resource Type` for each Work. Each work can be assigned its own `Display Label` (i.e. Title) and `Resource Type`, but all other metadata will be applied to all works created in the batch. To change the resource type of all files at once, select the desired resource type for the first file and click “Set all to this Resource Type”
+
+![Editing Display Labels and Resource Types](/images/screenshots/create-batch-works-3.png)
+
+### Describing Works
```

Comment:
is this really about batch editing MD for existing works? or when batch loading.

Reply:
changed alt text to clarify